android/app/build.gradle

```diff
@@ -93,7 +92,7 @@ android {
         }
         development {
             dimension "default"
-            applicationIdSuffix ".dev"
```

Comment:
Is it necessary to remove the applicationIdSuffix? 
The idea is that the different flavors production, development and staging can be installed simultaneously and removing the suffix seems to prevent that, doesn't it?